CRM/Prospect/Setup/AddProspectCategoryCgExtendsValue.php

```diff
@@ -23,12 +23,36 @@ public function apply() {
     civicrm_api3('OptionValue', 'create', [
       'option_group_id' => 'cg_extend_objects',
       'name' => 'civicrm_case',
-      'label' => $prospectCategoryLabel,
+      'label' => self::PROSPECTS_CATEGORY_LABEL,
       'value' => 'prospecting',
       'description' => 'CRM_Prospect_Helper_CaseTypeCategory::getProspectCaseTypes;',
       'is_active' => TRUE,
       'is_reserved' => TRUE,
     ]);
   }
 
+  /**
+   * Enables/Disables Prospects option values.
```

Comment:
Can you add in function description why we need to modify then description field?